src/components/forms/transaction/TransactionForm.tsx

```diff
@@ -157,6 +157,7 @@ async function onSubmit(data: FormValues, action: 'add' | 'update' | 'delete', o
     mutate(`/api/splits/${split.account.guid}`);
   });
 
+  mutate('/api/monthly-totals');
```

Comment:
new transactions affect monthly totals. Probably we can optimise by updating only the affected accounts but it's not easy because we have to propagate this total through all the parents. Maybe calling `getMonthlyTotals` query with only the accounts of a specific type would help a bit already.